src/components/OnboardingWidget/OnboardingWidget.test.tsx

```diff
@@ -0,0 +1,76 @@
+import { Button } from '@material-ui/core'
+import local from 'src/utils/storage/local'
+import { render, screen } from 'src/utils/test-utils'
+import { OnboardingWidget } from '.'
+
+describe('<OnboardingWidget>', () => {
+  test('renders widget on initial render and hides it after click on button', () => {
+    const text = 'New feature available!'
+
+    render(
+      <OnboardingWidget text={text} widgetLocalStorageId="someTestId">
+        <span>
+          <Button>Testbutton</Button>
+        </span>
```

Comment:
Are these `span`s needed because of the MUI `Tooltip`? If so, I'd make them a wrapper within the component itself.